build.gradle.kts

```diff
@@ -47,8 +48,15 @@ tasks.getByName<org.jetbrains.intellij.tasks.RunPluginVerifierTask>("runPluginVe
         "2023.1.4",
         "2023.1.5",
         "2023.2",
+        "2023.2.1",
+        "2023.2.2",
+        "2023.2.3",
+        "2023.2.4",
+        "2023.2.5",
+        "2023.3",
+        "2023.3.1",
     ))
-    verifierVersion.set("1.303")
+    verifierVersion.set("1.307")
```

Comment:
Per https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier-verifierversion this could be "LATEST"

Reply:
Bad idea. Previously, after the update, build was broken because of bug. 

This is the main reason for me to keep a fixed version and after each update clean everything and try to build/run plugin from scratch

Reply:
Ok